src/interpreter/explicitControlEvaluator.ts

```diff
@@ -162,6 +170,10 @@ const evaluators: AgendaItemEvaluatorMapping = {
     command: EmptyStatement,
     state: ExplicitControlEvaluatorState
   ) => {},
+  SequenceExpression: (
```

Comment:
Shall we move this down to be in lexicographical order?

Reply:
Oh yes, thanks for the catch!